namespaces/overlays/moc/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - mesh-for-data
```

Comment:
since we only include it for moc, should we just move the base files here, and leave it out of base? wdyt?

Reply:
Sure, will do

Reply:
second thought, I think this way maybe better, it keeps similar files closer together

Reply:
It make sense to me, I've moved it to the overlay. :+1: